third-party/rspamd/src/test/java/org/apache/james/rspamd/client/RSpamDHttpClientTest.java

```diff
@@ -113,7 +108,6 @@ void checkSpamMailUsingRSpamDClientWithExactPasswordShouldReturnAnalysisResultAs
         .then()
             .statusCode(HttpStatus.OK_200)
             .body("action", is(analysisResult.getAction().getDescription()))
-            .body("score", is(analysisResult.getScore()))
```

Comment:
As Tung said, these tests were green on his local. But CI failed. This means on Tung's local the merged input stream we send to RSpamD by our client is having the same score as the message body we sent manually to RSpamD through its web UI, but the CI is not the same case.
Plus with what we tested on diff machines, we likely can confirm that RSpamD (at least our docker image) spam scoring is diff on machines.

So I propose let disabling the scoring assertion for now. Let's say our client does the job regardless of RSpamD's scoring behavior and spend time investigating that behavior later.

What are your opinions?

Reply:
I would agree to that. Thanks for looking into this

Reply:
I agree with that too. 
We can check score > 0 (or is exist), it will be better than removing checking this property.